akka-stream-tests/src/test/java/akka/stream/javadsl/FlowTest.java

```diff
@@ -533,6 +533,61 @@ public String apply(String aggr, String in) throws Exception {
   }
 
   @Test
+  public void mustBeAbleToUseBatch() throws Exception {
+    final JavaTestKit probe = new JavaTestKit(system);
+    final List<String> input = Arrays.asList("A", "B", "C");
+    final Flow<String, String, BoxedUnit> flow = Flow.of(String.class).batch(3L, new Function<String, String>() {
```

Comment:
and we can happily use lambdas now—we just have not rewritten all the existing tests